src/world/block.c

```diff
@@ -622,7 +617,10 @@ WorldBlock** createWorldBlocks(UnkBoneFox* arg0, s32 skyBoxObjId, s32 blockMinOb
         s0->descriptor = ptr[i - num1];
         s0->prev = NULL;
         s0->next = NULL;
-        if (1) { } if (1) { } // TODO fake match
+        if (1) {
```

Comment:
I think it's good that these get split over multiple lines, but might be worth checking what`AllowShortIfStatementsOnASingleLine: WithoutElse` would do.